tools/test.py

```diff
@@ -161,20 +161,11 @@ def main():
             for ds_cfg in cfg.data.test:
                 ds_cfg.pipeline = replace_ImageToTensor(ds_cfg.pipeline)
 
-    if args.gpu_ids is not None:
-        cfg.gpu_ids = args.gpu_ids
-    else:
-        cfg.gpu_ids = range(1)
+    cfg.gpu_ids = [args.gpu_id]
```

Comment:
Do we need to change `cfg.gpu_ids` -> `cfg.gpu_id`?

Reply:
No, it is a list when distributed.